src/new.rs

```diff
@@ -30,7 +30,17 @@ impl New {
                 "__Cargo__.toml" => "Cargo.toml",
                 path => path,
             };
-            let content = content.replace("__project_name__", &self.project_name);
+            let mut content = content.replace("__project_name__", &self.project_name);
+            #[cfg(target_os = "windows")]
+            {
+                use regex::Regex;
+
+                content = Regex::new("[^\r]\n").unwrap().replace_all(&content, "\r\n").to_string();
```

Comment:
actually, change the regex to
[^\r]\n$

Reply:
I'm not sure that makes sense or is necessary.  \r\n is the end of the line, that's built into the definition.

Reply:
You can still format!("\r\n") and that would randomly get replaced by "\n" on linux, which can break templates. Same thing the other way around format!("line 1 \n\n line2").